src/components/ebay-textbox/test/test.browser.js

```diff
@@ -41,6 +41,10 @@ describe('given an input textbox with floating label and no value', () => {
         component = await render(template, input);
     });
 
+    it('then component is wrapped into floating label element', () => {
+        expect(component.getByTestId('floating-label')).has.class('floating-label');
```

Comment:
I know as a rule we try to not test by TestIds. I ran into the same issue. 
I would try to do `getByText` on ``floatingLabel` and then check the parent if it has floating-label.
You can also probably check the root element has `floatingLabel` as well using `component.container.firstElementChild` or something like that,

Reply:
Cool, done. Out of curiosity, why don't you use test-id's?